views/hud.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta name="apple-mobile-web-app-capable" content="yes">
+  
+	<title>Nightscout</title>
+  
+	<link href="/images/round1.png" rel="icon" id="favicon" type="image/png" />
+	<link rel="apple-touch-icon" sizes="57x57" href="/images/apple-touch-icon-57x57.png">
+	<link rel="apple-touch-icon" sizes="60x60" href="/images/apple-touch-icon-60x60.png">
+	<link rel="apple-touch-icon" sizes="72x72" href="/images/apple-touch-icon-72x72.png">
+	<link rel="apple-touch-icon" sizes="76x76" href="/images/apple-touch-icon-76x76.png">
+	<link rel="apple-touch-icon" sizes="114x114" href="/images/apple-touch-icon-114x114.png">
+	<link rel="apple-touch-icon" sizes="120x120" href="/images/apple-touch-icon-120x120.png">
+	<link rel="apple-touch-icon" sizes="144x144" href="/images/apple-touch-icon-144x144.png">
+	<link rel="apple-touch-icon" sizes="152x152" href="/images/apple-touch-icon-152x152.png">
+	<link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-180x180.png">
+  
+	<style type="text/css">
+		@import url("//fonts.googleapis.com/css?family=Open+Sans:700");
+    	body {
+			text-align: center;
+			margin: 0 0;
+			padding: 0;
+			overflow: hidden;
+			font-family: 'Open Sans';
+			color: white;
+			background-color: black;
+			-moz-transform: scale(1, -1);
+			-webkit-transform: scale(1, -1);
+			-o-transform: scale(1, -1);
+			-ms-transform: scale(1, -1);
+			transform: scale(1, -1);
+		}
+		main {
+			display: -webkit-box;
+			display: -ms-flexbox;
+			display: -webkit-flex;
+			display: flex;
+			-webkit-box-align: center;
+			-ms-flex-align: center;
+			-webkit-align-items: center;
+			align-items: center;
+			height: 100vh;
+		}
+		.inner {
+			width: 100%;
+			-webkit-transform: translateY(-2%);
+		}
+		#trend {
+			display: -ms-flexbox;
+			display: -webkit-flex;
+			display: flex;
+			-ms-flex-align: center;
+			-webkit-align-items: center;
+			align-items: center;
+			justify-content: center;
+			-webkit-flex-direction: row;
+			flex-direction: row;
+		}
+		#bgnow, #arrowDiv {
+			display: flex;
+			flex-grow: 0;
+			font-weight: 700;
+			font-size: 30vmin;
+			padding: 0 20px;
+			margin: 0;
+		}
+		img#arrow {
+			height: 15vmin;
+		}font-size: 15vmin;
+		}
+		.stale {
+			text-decoration: line-through;
+		}
+  </style>
+</head>
+
+<body>
+
+	<main>
+		<div class="inner">
+			<div id="trend">
+				<div id="bgnow"></div>
+				<div id="arrowDiv"><img id="arrow" src=""/></div>
+			</div>
+		</div>
+	</main>
+	
+	<script src="/api/v1/status.js"></script>
```

Comment:
append a unique identifier such as '/api/v1/status.json?t=' + new Date().getTime(); to prevent cache issues, similar to https://github.com/nightscout/cgm-remote-monitor/blob/7c272ec67a7795e0f4edf7ff9abdfac1bd1308a0/lib/client/index.js#L46